plugins/woocommerce/i18n/states.php

```diff
@@ -321,6 +321,41 @@
 		'CN31' => __( 'Tibet / 西藏', 'woocommerce' ),
 		'CN32' => __( 'Xinjiang / 新疆', 'woocommerce' ),
 	),
+	'CO' => array( // Colombia States.
+		'CO-DC' => __( 'Distrito Capital de Bogotá', 'woocommerce' ),
+		'CO-AMA' => __( 'Amazonas', 'woocommerce' ),
+		'CO-ANT' => __( 'Antioquia', 'woocommerce' ),
+		'CO-ARA' => __( 'Arauca', 'woocommerce' ),
+		'CO-ATL' => __( 'Atlántico', 'woocommerce' ),
+		'CO-BOL' => __( 'Bolívar', 'woocommerce' ),
+		'CO-BOY' => __( 'Boyacá', 'woocommerce' ),
+		'CO-CAL' => __( 'Caldas', 'woocommerce' ),
+		'CO-CAQ' => __( 'Caquetá', 'woocommerce' ),
+		'CO-CAS' => __( 'Casanare', 'woocommerce' ),
+		'CO-CAU' => __( 'Cauca', 'woocommerce' ),
+		'CO-CES' => __( 'Cesar', 'woocommerce' ),
+		'CO-COR' => __( 'Córdoba', 'woocommerce' ),
+		'CO-CUN' => __( 'Cundinamarca', 'woocommerce' ),
+		'CO-CHO' => __( 'Chocó', 'woocommerce' ),
+		'CO-GUA' => __( 'Guainía', 'woocommerce' ),
+		'CO-GUV' => __( 'Guaviare', 'woocommerce' ),
+		'CO-HUI' => __( 'Huila', 'woocommerce' ),
+		'CO-LAG' => __( 'La Guajira', 'woocommerce' ),
+		'CO-MAG' => __( 'Magdalena', 'woocommerce' ),
+		'CO-MET' => __( 'Meta', 'woocommerce' ),
+		'CO-NAR' => __( 'Nariño', 'woocommerce' ),
+		'CO-NSA' => __( 'Norte de Santander', 'woocommerce' ),
+		'CO-PUT' => __( 'Putumayo', 'woocommerce' ),
+		'CO-QUI' => __( 'Quindío', 'woocommerce' ),
+		'CO-RIS' => __( 'Risaralda', 'woocommerce' ),
+		'CO-SAP' => __( 'San Andrés', 'woocommerce' ),
```

Comment:
According to CLDR, this should be [San Andrés & Providencia](https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L907).